Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
-gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: 'datadog-5.0.0'
+gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: ENV['OMNIBUS_RUBY_BRANCH']
```

Comment:
To work well with the agent build I think it's safer to use a default value for `OMNIBUS_RUBY_BRANCH` in the entrypoint script here: https://github.com/DataDog/dd-agent-omnibus/blob/master/omnibus_build.sh#L13

Also helps when running the build containers locally.